src/client/http_client.rs

```diff
@@ -664,23 +665,48 @@ impl QueryRequest {
                 scope,
                 timeout,
                 transaction,
+                parameters,
             } => {
-                let mut json = HashMap::new();
+                let mut json: HashMap<String, serde_json::Value> = HashMap::new();
```

Comment:
Why did we have to change this to be annotated?

Reply:
We don't I think I just did it while getting things working, will change now